config/bootscripts/boot-odroid-xu4-next.cmd

```diff
@@ -1,93 +1,24 @@
-# This is an example file to generate boot.scr - a boot script for U-Boot
```

Comment:
probably I should use single quotes & embedded env variables
setenv prefix '/boot'